src/writing-your-own-application/no-std-applications/understanding-esp-template.md

```diff
@@ -0,0 +1,198 @@
+# Understanding esp-template
+
+Now that we know how to [generate a no_std project], let's inspect what the generated
+project contains and try to understand every part of it.
+
+## Inspecting the generated Project
+
+When creating a project from [esp-template] using the `esp32c3` target without dev-container support and without support for memory allocations, should generate a file structure like this:
+
+```text
+│   .gitignore
+│   Cargo.toml
+│   LICENSE-APACHE
+│   LICENSE-MIT
+│   rust-toolchain.toml
+│
+├───.cargo
+│       config.toml
+│
+├───.vscode
+│       settings.json
+│
+└───src
+        main.rs
```

Comment:
```suggestion
├─ .cargo/
│  ├─ config.toml
├─ .vscode/
│  ├─ settings.json
├─ src/
│  ├─ main.rs
├─ .gitignore
├─ Cargo.toml
├─ LICENSE-APACHE
├─ LICENSE-MIT
├─ rust-toolchain.toml
```

This block just renders kind of weird in its current form, my suggestion (IMO) looks a bit better? Feel free to ignore.